app/src/main/java/com/github/se/wanderpals/ui/navigation/Destination.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.se.wanderpals.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountBox
+import androidx.compose.material.icons.filled.DateRange
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.material.icons.filled.ShoppingCart
+import androidx.compose.ui.graphics.vector.ImageVector
+
+/**
+ * Data class representing a destination in the app.
+ *
+ * @param route The route of the destination.
+ * @param icon The icon of the destination.
+ * @param text The text of the destination.
+ */
+data class Destination(val route: String, val icon: ImageVector, val text: String)
+
+/** List of top level destinations in the app. */
+val TOP_LEVEL_DESTINATIONS =
```

Comment:
You should modify the icon parameter because it displays the same thing, maybe find pics on internet to represent the different icon

Reply:
I'm going to remove it to make the code more compact for now then, since it's not currently used and I don't see any potential use from it